testing/tests/DevExpress.ui.widgets/popover.tests.js

```diff
@@ -337,6 +338,28 @@ QUnit.module('options change', () => {
             fixtures.collisionTopLeft.drop();
         }
     });
+
+    QUnit.test('show warning if deprecated \'elementAttr\' option is used', function(assert) {
+        $('body').append('<div id="popover"></div>');
```

Comment:
why do u append new element to the body? Let's just init popover on `$('#what')` element

Reply:
done

Reply:
done